plugin/trino-phoenix5/src/test/java/io/trino/plugin/phoenix5/TestPhoenixConnectorTest.java

```diff
@@ -651,6 +652,21 @@ protected void verifyConcurrentAddColumnFailurePermissible(Exception e)
                 .hasMessageContaining("Concurrent modification to table");
     }
 
+    @Override
+    public void testCreateTableWithLongTableName()
+    {
+        // TODO: Find the maximum table name length in Phoenix and implement this test.
+        // Table name length with 65536 chars throws "startRow's length must be less than or equal to 32767 to meet the criteria for a row key."
+        // 32767 chars still causes the same error and shorter names (e.g. 10000) causes timeout.
+        throw new SkipException("TODO");
+    }
+
+    @Override
+    protected OptionalInt maxTableNameLength()
+    {
+        return OptionalInt.of(32767);
```

Comment:
remove this and `MAX_TABLE_NAME_LENGTH = 32767;` in the client  if the TODO above stays

Reply:
remove `maxTableNameLength()` implementation